satellite/marketing/marketingweb/server.go

```diff
@@ -186,11 +190,23 @@ func formToStruct(w http.ResponseWriter, req *http.Request) (o marketing.NewOffe
 	return o,nil
 }
 
+func (s *Server) reRenderTables(w http.ResponseWriter){
+    req,err := http.NewRequest("GET","/",nil)
+    if err != nil {
+        s.log.Error("Rerender tables err", zap.Error(err))
+        s.serveInternalError(w,req,err)
+        return
+    }
+
+    http.Redirect(w,req,"/",http.StatusFound)
```

Comment:
Use `http.StatusSeeOther` to prevent propagating `POST`.

Reply:
I was originally re-assigning the method type for the request in the line before (i.e. add `req.Method = "GET"` before `http.Redirect(....`) and that would also prevent propagating a POST and I was looking for a way to do it on one line but I missed `http.StatusSeeOther`. Will use that instead.